### DISCUSSION TITLE
Bottom notification text is unreadable in dark mode

### DISCUSSION THREAD

**Discussion by dar-kod** [NightscoutFoundation/xDrip#2920] [General]
Is this a phone specific issue, or do you have this problem in dark mode as well? It is very hard to read.

![photo_2023-07-02_21-12-00](https://github.com/NightscoutFoundation/xDrip/assets/24446806/3e6a1f40-e056-4c5f-ab08-c28a2dd75ae2)
![photo_2023-07-02_21-12-29](https://github.com/NightscoutFoundation/xDrip/assets/24446806/22e8ff5c-673f-4b3b-8a2a-c2c14db1b9b6)

**Comment by Navid200** [comment]
You can change most of the colors (not all).
https://navid200.github.io/xDrip/docs/Legend.html

Have you tried that?

**Comment by dar-kod** [comment]
I have never changed the default colors, and it seems that this color change is not available there, because this color is not available anywhere in "xDrip+ Color Settings".

**Reply by Navid200** [reply]
I plan to fix this.
